packages/core/__tests__/cli/build.test.ts

```diff
@@ -1614,7 +1664,7 @@ describe('CLI: --build --dry', () => {
           let buildResult = await projects.root.buildDeclaration({ flags: ['--dry'] });
           expect(buildResult.exitCode).toBe(0);
           expect(stripAnsi(buildResult.stdout)).toMatch(
-            `A non-dry build would build project '${projects.main.filePath('tsconfig.json')}'`,
+            ` A non-dry build would update timestamps for output of project '${projects.main.filePath('tsconfig.json')}'`,
```

Comment:
This also seems to be a change/optimization within TSC that's smart enough to realize root project would have no changed output and hence would only update timestamps (?)